File: android/src/main/java/expo/modules/moduleapptododavi/AvatarModule.kt
package expo.modules.moduleapptododavi

import expo.modules.kotlin.modules.Module
import expo.modules.kotlin.modules.ModuleDefinition

class AvatarModule : Module() {
  override fun definition() = ModuleDefinition {
    Name("AvatarView")

    View(AvatarView::class) {
      Prop("name") { view: AvatarView, name: String ->
        view.setName(name)
      }
    }
  }
}

File: ios/AvatarModule.swift
import ExpoModulesCore

public class AvatarModule: Module {
  public func definition() -> ModuleDefinition {
    Name("AvatarView")

    View(AvatarView.self) {
      Prop("name") { (view: AvatarView, name: String) in
        view.name = name
      }
    }
  }
}
